lblod/ember-rdfa-editor-lblod-plugins
# GN-4404: make number-variable placeholder consistent

### Overview
This ensures that the placeholder used for the number variable is consistent (both in the dynamic and static view).

##### connected issues and PRs:
Solves https://binnenland.atlassian.net/browse/GN-4404?atlOrigin=eyJpIjoiM2E1ZjM4NTg1OWFiNDUyODk4ZDNhZjhkNmZlYmE5ZGEiLCJwIjoiaiJ9 partially

### How to test/reproduce
- Switch your browser language to dutch
- Insert a number, notice that the placeholder states `Voeg getal in`
- View the exported html of the document. Notice that here the placeholder also states `Voeg getal in`.

### Challenges/uncertainties
- Currently the `Voeg getal in` string for the static version of a number variable is hard encoded in dutch. This will be solved once we take the document-language into account. 
- There are other variable nodes which still contain untranslated placeholder, these will need to be fixed once we take the document-language into account.
- The other issue stated in [GN-4404](https://binnenland.atlassian.net/browse/GN-4404?atlOrigin=eyJpIjoiM2E1ZjM4NTg1OWFiNDUyODk4ZDNhZjhkNmZlYmE5ZGEiLCJwIjoiaiJ9), where raw html was exported, could not be reproduced.



### Checks PR readiness
